src/plugins/intel_gpu/src/kernel_selector/jitter.cpp

```diff
@@ -1489,6 +1489,16 @@ JitConstants MakeTypeJitConstants(Datatype dataType, const std::string& macroNam
             type_size = "0.5f";
             is_fp = false;
             break;
+        case Datatype::BF16:
+            type = "ushort";
+            val_one = "(ushort) 1";
+            val_zero = "(ushort) 0";
+            to_type = "_convert_bfloat16_as_ushort(v)";
+            to_type_sat = "_convert_bfloat16_as_ushort(v)";
+            as_type = "_convert_bfloat16_as_ushort(v)";
```

Comment:
behavior of as_type is to keep the current bits as-is. But it is changing bits. I think it is not an expected behavior..

Reply:
deleted as_type definition at the moment